website/pages/docs/plugins/sources/overview.mdx

```diff
@@ -31,9 +31,9 @@ Official source plugins follow [release stages](source-plugin-release-stages).
 |--------------------------------------------------------------|
 | [Yandex](https://github.com/yandex-cloud/cq-provider-yandex) |
 
-# Source Plugin Release Stages
+## Source Plugin Release Stages
 
-[Official source plugins](sources) go through two release stages: Preview, and GA.
```

Comment:
Similar to release stages, `sources` used to be a separate page. The fix is to link to the anchor